README.md

```diff
@@ -18,6 +18,30 @@ git clone https://github.com/redhat-et/copilot-ops.git
 # Build the binary
 make build
 ```
+To install the command line tool, complete the following steps:
+
+```console
+# Create bin directory
+mkdir -p ${HOME}/bin
+
+# Save binary to bin directory (run from copilot-ops directory)
+cp copilot-ops ${HOME}/bin
```

Comment:
for scripting I use [install](https://man7.org/linux/man-pages/man1/install.1.html) instead of cp to copy binaries and it already sets the mode to executable. however I would add a trailing slash to make sure bin is treated as the target dir and not target file if mkdir was skipped by accident.